---
title: LOCALDB_ERROR_CANNOT_ACCESS_INSTANCE_FOLDER | Документация Майкрософт
ms.custom: ''
ms.date: 03/04/2017
ms.prod: sql
ms.reviewer: ''
ms.technology: performance
ms.topic: reference
ms.assetid: c4288265-43f7-44fd-acab-ba64d04264d9
author: stevestein
ms.author: sstein
ms.openlocfilehash: d0354ba6d980b359778a66fc13b65fe233cacba1
ms.sourcegitcommit: b87d36c46b39af8b929ad94ec707dee8800950f5
ms.translationtype: MT
ms.contentlocale: ru-RU
ms.lasthandoff: 02/08/2020
ms.locfileid: "67995994"
---
# <a name="localdb_error_cannot_access_instance_folder"></a>LOCALDB_ERROR_CANNOT_ACCESS_INSTANCE_FOLDER
[!INCLUDE[appliesto-ss-xxxx-xxxx-xxx-md](../../includes/appliesto-ss-xxxx-xxxx-xxx-md.md)]
    
## <a name="details"></a>Сведения  
  
|||  
|-|-|  
|Название продукта|SQL Server|  
|Идентификатор события|261|  
|Источник события|Среда выполнения локальной базы данных SQL Server 12.0|  
|Компонент|API среды выполнения локальной базы данных|  
|Текст сообщения|Не удается получить доступ к папке экземпляра локальной базы данных:%% LOCALAPPDATA%% \\\ Microsoft\Microsoft SQL Server локальное имя\>экземпляра дб\инстанцес<.|  
  
## <a name="explanation"></a>Объяснение  
 Не удалось получить доступ к папке экземпляра.  
  
## <a name="user-action"></a>Действие пользователя  
  
